src/kudu/consensus/raft_consensus.cc

```diff
@@ -2868,6 +2898,7 @@ void RaftConsensus::CompleteConfigChangeRoundUnlocked(ConsensusRound* round, con
       LOG_WITH_PREFIX_UNLOCKED(INFO) << "Aborting config change with OpId "
                                      << op_id << ": " << status.ToString();
       cmeta_->clear_pending_config();
+      CHECK_OK(routing_table_->UpdateRaftConfig(cmeta_->ActiveConfig()));
```

Comment:
nit:  adding a comment that config abort needs routing table to be updated/reverted

Reply:
Done